components/Faq/Faq.tsx

```diff
@@ -1,36 +1,72 @@
-import { Container, Stack } from "../bootstrap"
+import { Container, Stack, Col } from "../bootstrap"
 import { FaqQandA } from "./FaqQandA"
 import { FaqCard } from "./FaqCard"
 import styles from "./Faq.module.css"
-import content from "./faqContent.json"
+import { faqContentCards } from "./faqContent"
+import { DropdownButton } from "../shared/DropdownButton/DropdownButton"
+import { useMediaQuery } from "usehooks-ts"
+import { stylingProps } from "../shared/DropdownButton/DropdownButton"
 
 export const Faq = () => {
-  const faqData: FaqData = content
-  const faqKeys: string[] = Object.keys(faqData)
+  const mdMobile = useMediaQuery("(max-width: 768px)")
+  const xsMobile = useMediaQuery("(max-width: 418px)")
 
-  interface FaqData {
-    [key: string]: string | FaqDataCard | any
+  const faqCardHeadings: Array<string> = faqContentCards.map(key => {
+    return key.heading
+  })
+
+  interface Headings {
+    id: number
+    itemName: string
+  }
+
+  interface FaqContentCards {
+    [key: string]: string | FaqContentCard | any
   }
 
-  interface FaqDataCard {
+  interface FaqContentCard {
+    id: number
     heading: string
     qAndA: { question: string; answer: string }[]
   }
 
+  const header1 = "FAQ"
+  const dropdownTitle = "All FAQ Topics"
+
+  const styling: stylingProps = {
+    width: { desktop: "418px", mobile: xsMobile ? "100%" : "418px" }
+  }
+
   return (
-    <>
-      <Container fluid className={styles.faqLayout}>
-        <h1>FAQ</h1>
-        <Stack direction="vertical" gap={4}>
-          {faqKeys.map((key, index) => (
-            <FaqCard
-              key={index}
-              heading={faqData[key].heading}
-              qAndAs={faqData[key].qAndA}
-            ></FaqCard>
-          ))}
-        </Stack>
-      </Container>
-    </>
+    <Container fluid className={styles.faqLayout}>
+      <Stack
+        className={styles.faqHeader}
+        direction={mdMobile ? "vertical" : "horizontal"}
+        gap={mdMobile ? 0 : 4}
+      >
+        <h1>{header1}</h1>
+        <Col xs={xsMobile ? 12 : !12}>
+          <DropdownButton
+            key={dropdownTitle}
+            title={dropdownTitle}
+            styling={styling}
+            variant="secondary"
+          >
+            {faqCardHeadings}
+          </DropdownButton>
+        </Col>
+      </Stack>
+
+      <Stack direction="vertical" gap={4}>
+        {faqContentCards.map(card => (
```

Comment:
how do you plan to filter this to the selected section in the dropdown, as in the figma mocks? The react bootstrap Dropdown component has an onSelect callback you can use to set state and then filter based on the selected value: https://react-bootstrap.netlify.app/docs/components/dropdowns#dropdown

Reply:
there is no filtering requirement in this dropdown. Did you take a look at the figma. It provides a set of jump links to navigate to different sections on the page.

Reply:
oh I see what you mean...I can commit changes...